### PR TITLE
Added cartPage features

### PR DESCRIPTION
**Pull request by vmjanani92**
Hi Karthik,

Please review the PR code.

Added orderPage
Added orderPageTest

### CODE REVIEW

File: src/Pages/CartPage.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Pages;
+
+public class CartPage {
+	
+		public static void doCart() {
+		
+		System.out.println("Cart Page");
```
Comment (unceasingwaves92) [inline src/Pages/CartPage.java:7]: Hi Janani,

Kindly following the below changes.
Added cart page logic

File: src/tests/CartPageTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package tests;
+
+public class CartPageTest {
+	
+		
+	public void cartTest() {
+		
+		System.out.println("true");
```
Comment (unceasingwaves92) [inline src/tests/CartPageTest.java:11]: Hi Janani,

where is order page actions

Comment (vmjanani92) [reply src/tests/CartPageTest.java:11]: Done. Logic action added.

File: src/Pages/CartPage.java (hunk 1 above)
Comment (vmjanani92) [reply src/Pages/CartPage.java:7]: Done karthik.

**Review by unceasingwaves92 — APPROVED** [review]
PR is approved

**Pull request merged**